openslides_backend/action/actions/user/save_saml_account.py

```diff
@@ -109,7 +124,16 @@ def validate_fields(self, instance_old: dict[str, Any]) -> dict[str, Any]:
                     if isinstance((tx := instance_old[payload_field]), list) and len(tx)
                     else tx
                 )
-                if value not in (None, []):
+                if payload_field == "gender":
```

Comment:
The requirement for the save_saml_account action is to create a gender-instance, if the value of the `gender` is not in the collection.
IMO this is the most convenient way to add new gender-entries to the collection and the same behavior of accepting all gender-values in save_saml_account as before.

Please update the action documentation and implement these feature as described.

Reply:
Done